keanu-project/src/main/java/io/improbable/keanu/distributions/continuous/Pareto.java

```diff
@@ -58,12 +56,11 @@ public DoubleTensor logProb(DoubleTensor x) {
     }
 
     public static DoubleVertex logProbOutput(DoublePlaceholderVertex x, DoublePlaceholderVertex location, DoublePlaceholderVertex scale) {
-        final DoubleVertex zero = ConstantVertex.of(0.);
-        final BooleanVertex paramsAreValid = location.greaterThan(zero)
-            .and(scale.greaterThan(zero));
-        paramsAreValid.assertTrue("Location and scale must be strictly positive");
-
-        final DoubleVertex invalidXMask = x.toLessThanOrEqualToMask(location);
+        final DoubleVertex invalidXMask = x.toGreaterThanMask(location)
```

Comment:
Nice trick.

Reply:
We should improve the semantics of masks with using the +/- approach instead of 0/1. Can you raise a ticket for it?

Reply:
Done.